### PR TITLE
Update IRC to Discord and make forking info more obvious

### PR DESCRIPTION
**Pull request by momijizukamori**

### CODE REVIEW

File: www/apply.shtml [hunk 1]
```diff
@@ -18,7 +18,7 @@
           <tr><th><label for="dwname">Your DW username (if you have one):</label></th><td><input type="text" id="dwname" name="dwname" size="25" class="dwuser stretch"></td></tr>
           <tr><th><label for="ljname">Your LJ username (if you have one):</label></th><td><input type="text" id="ljname" name="ljname" size="15" class="ljuser stretch"></td></tr>
           <tr><th><label for="ghname">Your GitHub username:</label></th><td><input type="text" id="ghname" name="ghname" size="25" class="stretch"> (<a href="/why-github.shtml">why do I need a GitHub username?</a>)</td></tr>
-          <tr><th><label for="ircname">Your IRC name (if you use it):</label></th><td><input type="text" id="ircname" name="ircname" size="25" class="stretch"></td></tr>
+          <tr><th><label for="ircname">Your Discord name (if you use it):</label></th><td><input type="text" id="discordname" name="discordname" size="25" class="stretch"></td></tr>
```
Comment (kareila) [inline www/apply.shtml:21]: Looks like `label for="ircname"` should also be updated to `discordname` too, just for consistency.

**Comment by Sophira** [comment]
Ah, sorry, you need to grep for both `ljusername` and `lj_username` if you want to do that, my mistake!

**Comment by momijizukamori** [comment]
Oh, thanks for the info @Sophira! In that case maybe we should just drop that field entirely, if it's not actually being used anywhere (as I don't feel like mucking around with DB migrations, lol)

**Comment by momijizukamori** [comment]
Okay, Jen pointed out that it's still useful to have the discord username even if we don't keep it in the DB, just for making it easier to contact people about issues with their application, so I fixed the label, and just remove the IRC data from /www/do/dreamhacks

**Comment by Sophira** [comment]
I should have mentioned something in my comment, sorry! The IRC nick field isn't *entirely* unused; people can still add it to the database after the fact by typing `edit-dreamhack-info` at the Dreamhack command line, which allows them to edit their details, and this includes being able to add an IRC nick afterwards; it's just that nobody did that, which is honestly not entirely surprising.

(It doesn't give *direct* access to the database; rather, it works by dropping a file into `/dreamhack/var/maintain.d/` which is picked up by the [`lib/bin/cron/maintain-dreamhacks`](https://github.com/dreamwidth/dreamhack/blob/master/lib/bin/cron/maintain-dreamhacks) script which runs as root via cron every minute, and that script is what actually makes the edits. I did it this way because the alternative would be allowing every user to have write access to everybody's info, which is obviously Not A Good Idea.)

So, if you want to completely remove that field, you'll also want to prevent updates to the field via these two scripts! I apologise for not mentioning this in my other comment.

**Comment by Sophira** [comment]
Is there anything I can do to help with this PR? I feel like I may have unintentionally discouraged people from working on this with my last response here and that's not what I intended to do. I'm happy to help however I can.

**Comment by kareila** [comment]
Thanks for pointing out that this hadn't been completed. I'm continuing this in #63.

**Pull request closed without merging**